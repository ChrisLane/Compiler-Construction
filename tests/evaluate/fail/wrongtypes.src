// We can only use operators on matching types. This should not evaluate.
main () {
    x = 1 == 1;
    x = $x + 2;

    b = 0;
    x = b +5;
}
