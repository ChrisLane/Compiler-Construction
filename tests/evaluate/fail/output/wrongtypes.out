Fatal error: exception Failure("Values are of different types. Operator cannot be applied.")
